dashboard/app/models/script_level.rb

```diff
@@ -258,7 +258,20 @@ def next_level_or_redirect_path_for_user(
       # to that lesson
       script_lesson_extras_path(script.name, (extras_lesson || lesson).relative_position)
     else
```

Comment:
Nit: you could make this `elsif end_of_lesson? && script.should_show_unit_overview_between_lessons && user.has_pilot_experiment?('end-of-lesson-redirects')` and eliminate one layer of nesting